pkgs/by-name/co/coc-pyright/package.nix

```diff
@@ -0,0 +1,27 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+}:
+
+buildNpmPackage {
+  pname = "coc-pyright";
+  version = "1.1.371";
```

Comment:
Looks like there is a new release, how about updating?
https://github.com/fannheyward/coc-pyright/commit/7a765863fab5b3345018377b47203e8cfb30b4e3